<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevaEstMonitoreoDialog</class>
 <widget class="QDialog" name="NuevaEstMonitoreoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Compañia minera:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboGrupo">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Unidad minera:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboUnidad">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Fotos</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLineEdit" name="txtPath1">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Primera foto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnFoto1">
          <property name="toolTip">
           <string>Cargar primera imagen</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLineEdit" name="txtPath2">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Segunda foto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnFoto2">
          <property name="toolTip">
           <string>Cargar segunda imagen</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLineEdit" name="txtPath3">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Tercera foto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnFoto3">
          <property name="toolTip">
           <string>Cargar tercera imagen</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Codigo de estacion:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtCodigo">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Ingrese aqui el codigo de estación</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Fecha:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="deFecha">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Hora:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="teHora">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="calendarPopup">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Descripcion:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPlainTextEdit" name="teDescripcion">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Datos de campo</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>PH:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="dsbPh">
          <property name="minimumSize">
           <size>
            <width>133</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>133</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Temp:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="dsbTemp">
          <property name="minimumSize">
           <size>
            <width>133</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>133</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>999.990000000000009</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>OD:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="dsbOd">
          <property name="minimumSize">
           <size>
            <width>133</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>133</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QFormLayout" name="formLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Cond:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="dsbCond">
          <property name="minimumSize">
           <size>
            <width>133</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>133</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>999999999.990000009536743</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Coordenadas</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Este:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="dsbEste">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <double>9999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Norte:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="dsbNorte">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <double>9999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Cota:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="dsbCota">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="suffix">
         <string>  (m.s.n.m)</string>
        </property>
        <property name="maximum">
         <double>9999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Descripción del punto</string>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/img/ui_-09-128.png</normaloff>:/img/ui_-09-128.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnGuardar">
       <property name="text">
        <string>Guardar</string>
       </property>
       <property name="icon">
        <iconset resource="rsc.qrc">
         <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="rsc.qrc">
         <normaloff>:/img/cancel.png</normaloff>:/img/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
